src/types/transfer.ts

```diff
@@ -30,6 +36,8 @@ export interface EnsureFundedParams extends SendTransactionParams {
   transactionParams?: SuggestedParams
   /** The (optional) transaction note, default: "Funding account to meet minimum requirement" */
   note?: TransactionNote
+  /** Whether to use the dispenser API to fund the account. Defaults to false. */
+  useDispenserApi?: boolean
```

Comment:
I think `fundingSource` allowing `'testnet-dispenser-api'` as a union type option is a cleaner way of doing it.

Reply:
@robdmoore closing as this is now taking a client instance